src/plugins/table/index.tsx

```diff
@@ -13,6 +13,7 @@ interface Props extends PluginProps {
   config: {
     maxRow?: number;
     maxCol?: number;
+    isShowDefaultText?: boolean;
```

Comment:
"is show" main focus on show/hide, but in this plugin, it add default text to generated texts.
Maybe "useDefaultText" or "addDefaultText" is better?